optuna/integration/sklearn.py

```diff
@@ -56,6 +57,24 @@
 _logger = logging.get_logger(__name__)
 
 
+def check_fit_params(
+    X,  # type: TwoDimArrayLikeType
+    fit_params,  # type: Dict
+    indices  # type: OneDimArrayLikeType
+):
```

Comment:
Could you annotate the return type?
Also, I suppose this method should be private (prefixed with `_`) as most users will not use this.